### DISCUSSION TITLE
article/af2e6f7b-0ef7-4249-b195-85a5be2791d2

### DISCUSSION THREAD

**Discussion by giscus** [siuze/NotionNext#4] [Announcements]
# article/af2e6f7b-0ef7-4249-b195-85a5be2791d2

2022年11月底至12月初，国内防疫政策调整后，各地都在经历这一特殊的过渡阶段，各地居民在2022年12月内经历新冠病毒感染结果阳性的情况十分普遍。通过分析微博这里社交媒体相关数据，有助于了解疫情在空间上和时间上的发展趋势，有助于关切社会舆论热点信息，对个人防疫卫生和政府决策都有一定帮助。

https://notion.siuze.top/article/af2e6f7b-0ef7-4249-b195-85a5be2791d2

**Comment by Guanchishan** [comment]
這數據真有意思！

**Comment by siuze** [comment]
我发现一个严重问题，爬取微博脚本设定的小时序列编写错误，漏了半夜23时（`range(0,24)`写成了`range(0,23)`），这导致部分数据缺漏了
